CRM/Utils/Check/Component/Schema.php

```diff
@@ -178,4 +178,19 @@ public function checkSmartGroupCustomFieldCriteria() {
     return $messages;
   }
 
+  public function checkMoneyValueFormatConfig() {
+    $messages = [];
+    if (CRM_Core_Config::singleton()->moneyvalueformat !== '%!i') {
+      $msg = new CRM_Utils_Check_Message(
+        __FUNCTION__,
+        ts('The Money Value format stored is deprecated please report your configuration on Gitlab'),
```

Comment:
Maybe add a link to gitlab

Reply:
Done and also done it in the Civi::log() call as well